src/index.css

```diff
@@ -25,3 +25,40 @@ code {
     'Courier New',
     monospace;
 }
```

Comment:
- [ ] It would be better if we delete the following files instead:
  - Src - `logo.svg`, `reportWebVitals.js`, `setupTest.js`
  - Public - `logo192.png`. `logo512.png`. `robots.txt`, `manifest.json`, and `favicon.ico`
![Screenshot from 2023-08-16 08-09-47](https://github.com/Nessrine88/MathMagicians/assets/46388113/aef8cf63-9f01-40ee-89ae-7d23ade56dd0)